diode-server/reconciler/mocks/client.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.43.2. DO NOT EDIT.
+// Code generated by mockery v2.42.1. DO NOT EDIT.
```

Comment:
maybe try to upgrade mockery to latest? but only cosmetic change

Reply:
That's odd, thought I was on the latest hence the change 🤨 Good spot